src/parsing-v2/index.spec.ts

```diff
@@ -160,6 +160,22 @@ describe("parsePaymentDestination validations", () => {
       }),
     )
   })
+
```

Comment:
I would add one more test to see if it can validate a lightning invoice with lightning protocol `lighting:[ln invoice]`.

Reply:
In fact this already exists https://github.com/GaloyMoney/galoy-client/blob/a53d00beba69b2538fa4c000e0c42c9cdb8e4a66/src/parsing-v2/index.spec.ts#L415